doc/nrf/ug_partition_manager.rst

```diff
@@ -0,0 +1,188 @@
+   When this property is set for a partition, it is required that the `span` property is also set.
+
+span: list
+   When creating sub_partitions, this option lists what partitions the sub_partitions should span across.
+   This property can not be used together with the `placement` property.
+
+.. _pm_yaml_preprocessing:
+
+Configuration File Preprocessing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each ``pm.yml`` file is preprocessed.
+Example usage of preprocessing is seen below:
+
+.. code-block:: yaml
+
+   #include <autoconf.h>
+   b0:
+     placement: {before: [mcuboot, app]}
+   #ifndef CONFIG_SOC_NRF9160
+   provision:
+     placement: last
+   #endif
+
+.. _pm_yaml_partition_types:
+
+Partition Types
+~~~~~~~~~~~~~~~
+
+It is required that each ``pm.yml`` defines one *image partition*.
+This is done by using the same name for the partition as the image name.
+
+All other partitions which have the `placement` property set are *placeholder
+partitions*.
+
+Partitions which have the `sub_partitions` property set are *phony partitions*,
+and do not occupy space in flash.
+
+In addition to the configuration provided in ``pm.yml``, each partition with the `placement` property set must have a corresponding size configuration in the images ``Kconfig`` file.
+The format of this configuration is as follows:
+
+.. code-block:: none
+
+   config PM_PARTITION_SIZE_[PARTITION_NAME]
+   hex "Flash space reserved for [partition name]."
+   default 0xD000
+   help
+     Flash space set aside for [partition name]. Note, the name
+     of this configuration needs to match the requirements set by the
+     script 'partition_manager.py'. See pm.yaml.
+
+.. _pm_build_system:
+
+Build system
+============
+This section describes how the Partition Manager is included by the Zephyr build system.
+
+If one or more sub-images are included in a build, a set of properties for that sub-image is appended to a global list.
+
+These properties are:
+   * Path to ``pm.yml``
+   * Build directory path
+   * Path to generated include folder
+
+Once CMake is done configuring the sub-images the Partition Manager script is executed in configure time (`execute_process`) with the aforementioned list as argument.
+The configurations generated by the Partition Manager script are imported as CMake variables. See :ref:`pm_generated_output_and_usage`
+
+.. _pm_generated_output_and_usage:
+
+Generated output and usage
+==========================
+For each sub-image and the root app, Partition Manager generates two files, one C header file and one Kconfig file.
+The C header file is used in the C code while the Kconfig file is imported in CMake.
+Both these files contain the start address and size of all partitions.
+The Kconfig file additionally contains the build directory and generated include folder for each image.
+
+C code usage
+   When Partition Manager is enabled, all source files are compiled with the define `USE_PARTITION_MANAGER` set to 1.
```

Comment:
``` suggestion
When Partition Manager is enabled, all source files are compiled with the define ``USE_PARTITION_MANAGER`` set to 1.
```

Reply:
This indentation is needed to get the formatting correct.

Reply:
sorry, my mistake